modules/build/src/main/scala/scala/build/internal/NativeBuilderHelper.scala

```diff
@@ -4,10 +4,11 @@ import java.math.BigInteger
 import java.security.MessageDigest
 
 import scala.build.Build
-import scala.scalanative.{build => sn}
 
 object NativeBuilderHelper {
 
+  case class SNCacheData(val changed: Boolean, val projectSha: String)
```

Comment:
NIT No need of `val`s in case classes:
```suggestion
  case class SNCacheData(changed: Boolean, projectSha: String)
```